__tests__/app.test.js

```diff
@@ -25,16 +23,42 @@ describe("3 api/categories", () => {
         });
       });
   });
-  
+  test("GET 200 - Returns an array of review objects with '`owner` which is the `username` from the users table`title`,`review_id`, `category`,`review_img_url`,`created_at`,`votes` ,`designer`,`comment_count` which is the total count of all the comments with this review_id", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then((response) => {
+        expect(response.body).not.toHaveLength(0);
+        response.body.forEach((review) => {
+          let dateVal = 0;
+          expect(review).toMatchObject({
+            owner: expect.any(String),
+            title: expect.any(String),
+            review_id: expect.any(Number),
+            category: expect.any(String),
+            review_img_url: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            designer: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+          const checkDate = Date.parse(review.created_at) > dateVal;
```

Comment:
This is ok but there is a jest matcher that can be used that would be more readable and would have saved you some time.